doc/releases/changelog-dev.md

```diff
@@ -258,6 +258,10 @@
 
 <h3>Bug fixes 🐛</h3>
 
+* `_try_wrap_in_custom_ctrl_op` in `pennylane/ops/op_math/controlled.py` now removes the input `op` from `QueuingManager`
+  before calling `ControlledQubitUnitary`, to avoid a [known bug](https://github.com/PennyLaneAI/catalyst/issues/1494) found in Catalyst.
+  [(#6926)](https://github.com/PennyLaneAI/pennylane/pull/6926)
+
```

Comment:
I would perhaps just add this changelog entry to the `ControlledQubitUnitary` changelog entry. It's basically a follow up to that PR.

Reply:
resolved at https://github.com/PennyLaneAI/pennylane/pull/6926/commits/486ab0f42db71c022b90f23c8fe7f3797791f3a9